search-commons/src/test/java/no/unit/nva/search/OpensearchTest.java

```diff
@@ -86,10 +89,16 @@ static void setUp() {
     }
 
     @AfterAll
-    static void afterAll() {
+    static void afterAll() throws Exception {
+        indexingClient.deleteIndex(indexName);
```

Comment:
Since there was a method which does not need to add documents to an index (and hence getting error), I have to create nested class and add all other methods in the nested class. This way we add  @beforeEach and  @afterEach. Test execution becomes also more readable.